docs/wiki/guides/current/SoftSerial.md

```diff
@@ -0,0 +1,254 @@
+
+### Softserial with HD OSD (displayport) over MSP
+
+This is not recommended, though it _may_ work. Both a SoftserialTx and an Softserial Rx pin must be defined. The data rate with anything but the simplest OSD would likely overwhelm the connection. Note that softserial is limited to 19200 baud so don't use rates above that.
+
+### Enabling Softserial
+
+Softserial requires custom CLI commands. A unused pin with an associated timer must be found, where the original purpose no longer is needed. The most common are PPM pins or LED_STRIP pins. Not all such pins will work on all boards, and you may need to do some reseach about using softserial on your board before you can make it work.
+
+First go to the CLI and type `resource`, to get a list of the currently assigned pin numbers for each of the currently available pads on the device.
+
+If, for instance, a `LED_STRIP 1` pin is defined to `A15`, and there is a LedStrip pad on the board, and you are not using a led strip, you could de-assign that pin from `LED_STRIP 1` with the CLI command:
+
+`resource led_strip 1 none`
+
+Then the pin can be assigned to the Softserial Port 1 Tx pin with:
+
+`resource serialTx 11 A15`
+
+If the resource reassignment is successful, a `diff` command in CLI will show:
+
+```
+resource LED_STRIP 1 NONE
+resource SERIAL_TX 11 A15
+```
+
+Note:
+
+- `serialTx` and `serialRx` settings of 11 and 12 are used to configure pins to Softserial ports 1 and 2 respectively.
+- a `diff` command in CLI will show changes from the default resource configuration
+- `resource show all` in CLI will provide a full list of all serial ports, timers etc.
+- just because the resources are re-assigned, it does not mean that the Softserial port will work properly.
+
+Oscarliang provides a good summary of [how to set up softserial](https://oscarliang.com/betaflight-soft-serial/).
+
+### The CLI Serial command.
+
+When Softserial is active, the `serial` CLI command will show the configured serial ports, and a line starting with 30, for Softserial port 1, or 31, for Softserial port 2, will appear.
+
+For example, if Softserial is configured for smartAudio on port 1, a line like this should appear when the `serial` command is sent:
+
+```
+serial 30 2048 115200 57600 0 115200
+```
+
+For more information, see the 'Serial' page in the Development section.
+
+### Historical notes:
+
+The following notes are quite old, and may not be relevant to current versions of Betaflight
+
+CLI commands to assign softserial ports 1 and 2:
+
+`resource serial_tx 11 <pin>`
```

Comment:
See above

Reply:
see above

Reply:
OK sorry it now says more clearly what the command to use in 4.5 is vs 4.4 and earlier.